demo/firebase-idtoken-flow-example/src/App.tsx

```diff
@@ -292,6 +295,35 @@ function App() {
     uiConsole(receipt);
   };
 
+  const recoverSecurityQuestionFactor = async (answer : string) => {
+    if (!coreKitInstance) { 
+      throw new Error("coreKitInstance is not set");
+    }
+    await securityQuestion.recoverSecurityQuestionFactor(coreKitInstance, answer); 
+  }
+
+  const createSecurityQuestion = async ( question: string, answer: string ) => {
+    if (!coreKitInstance) { 
+      throw new Error("coreKitInstance is not set");
+    }
+    await securityQuestion.setSecurityQuestion(coreKitInstance, question, answer);
+  }
+
+  const changeSecurityQuestion = async ( question: string, answer: string) => {
+    if (!coreKitInstance) { 
+      throw new Error("coreKitInstance is not set");
+    }
+    await securityQuestion.setSecurityQuestion(coreKitInstance, question, answer);
```

Comment:
in sdk we have a seperate function for changing security question, why we are not using that here ? and rather we are using setSecurityQuestion.